docs/release_guidelines.md

```diff
@@ -0,0 +1,14 @@
+# OCS-CI Releases
```

Comment:
You are missing to link this document in index to be part of the generated documentation.
https://github.com/red-hat-storage/ocs-ci/blob/master/docs/index.rst